Content.Server/GameObjects/Components/Engineering/SpawnAfterInteractComponent.cs

```diff
@@ -13,8 +15,8 @@ public class SpawnAfterInteractComponent : Component
         public override uint? NetID => ContentNetIDs.SPAWN_AFTER_INTERACT;
 
         [ViewVariables]
-        [DataField("prototype")]
-        public string? Prototype { get; private set; }
+        [DataField("prototype", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+        public string? Prototype;
```

Comment:
Same as above.

Reply:
when doing `field:` you can even omit the setter completely